Sources/Credentials/MultiTypeSafeCredentials.swift

```diff
@@ -22,7 +22,7 @@ import Foundation
 // MARK TypeSafeMultiCredentials
 
 /**
- A `TypeSafeMiddleware` for authenticating users using multiple authentication methods. A type conforming to `TypeSafeMultiCredentials` must implement a static array of `TypeSafeCredentials` types and an initializer, which takes a `TypeSafeCredentials` instance. The route will attempt to authenticate by iterating through this array of `TypeSafeCredentials` until an authentication succeeds. This returns an instance of the succesful `TypeSafeCredentials` which is used to initialize the `TypeSafeMultiCredentials` instance. If a plugin fails or you reach then end of your `TypeSafeCredentials array` an unauthorized response is sent. The type conforming to `TypeSafeMultiCredentials` can then be used as a middleware in codable routes.
+ A `TypeSafeMiddleware` protocol for using multiple authentication methods on a Codable route. An object conforming to this protocol, must contain a static array of the acceptable `TypeSafeCredentials` types and be initializable from the authentication instance that succeeded. If an authentication fails or you reach then end of your array, an unauthorized response is sent.
```

Comment:
Typo: then end of your array -> **the** end of your array
Also, unnecessary comma after "conforming to this protocol,"